includes/Core/Storage/Data_Encryption.php

```diff
@@ -83,19 +83,26 @@ public function encrypt( $value ) {
 	 * @return string|bool Decrypted value, or false on failure.
 	 */
 	public function decrypt( $raw_value ) {
-		if ( ! extension_loaded( 'openssl' ) ) {
+		if ( ! extension_loaded( 'openssl' ) || ! is_string( $raw_value ) ) {
 			return $raw_value;
 		}
 
-		$raw_value = base64_decode( $raw_value, true );
+		$decoded_value = base64_decode( $raw_value, true );
+
+		if ( false === $decoded_value ) {
+			// phpcs:ignore
```

Comment:
We shouldn't need this anymore, but please use the per-line ignore for the specific sniff rather than this form which ignores everything.